langroid/embedding_models/models.py

```diff
@@ -23,6 +25,22 @@ class OpenAIEmbeddingsConfig(EmbeddingModelsConfig):
     context_length: int = 8192
 
 
+class AzureOpenAIEmbeddingsConfig(EmbeddingModelsConfig):
+    model_type: str = "azure-openai"
+    model_name: str = "text-embedding-ada-002"
+    api_key: str = ""
+    azure_endpoint: str = ""
```

Comment:
can you please change it to "api_base" - to align with the AzureConfig class?

Reply:
the choice of args naming comes from the original Python SDK.
cc: @pchalasani 
ref: https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/embeddings?tabs=python-new

Reply:
I agree with @alexagr to keep the names consistent with AzureConfig

Reply:
@AbhishekRP2002 please make this `api_base` to align with `AzureCongig`